README.md

```diff
@@ -3,8 +3,7 @@
 ## Introduction
```

Comment:
[Set up the container runtime](https://github.com/openshift/windows-machine-config-operator/blob/6e5858b8bb64eee88928f62da8da6033df485a52/docs/vsphere-golden-image.md#4-set-up-the-container-runtime) section may need an update too?

Reply:
Good catch @jrvaldes I think we need to remove that section. From now on WMCO will manage the runtime so we don't need that part.